kcc-schema.json

```diff
@@ -0,0 +1,79 @@
+{
+   "$schema":"https://json-schema.org/draft/2020-12/schema",
+   "type":"object",
+   "properties":{
+      "credentialSubject":{
+         "type":"object",
+         "properties":{
+            "id":{
+               "type":"string",
+               "pattern":"^did:[a-z0-9]+:[a-zA-Z0-9.-_:]+$"
+            },
+            "countryOfResidence":{
+               "type":"string",
+               "pattern":"^[A-Z]{2}$"
+            },
+            "tier":{
+               "type":"string"
+            }
+         },
+         "required":[
+            "id",
+            "countryOfResidence"
+         ]
+      },
+      "issuer":{
+         "type":"string",
+         "pattern":"^did:[a-z0-9]+:[a-zA-Z0-9.-_:]+$"
+      },
+      "issuanceDate":{
+         "type":"string",
+         "format":"date-time"
+      },
+      "expirationDate":{
+         "type":"string",
+         "format":"date-time"
+      },
+      "credentialSchema":{
+         "type":"object",
+         "properties":{
+            "id":{
+               "type":"string",
+               "format":"uri"
+            },
+            "type":{
+               "type":"string",
+               "const":"JsonSchema"
+            }
+         },
+         "required":[
+            "id",
+            "type"
+         ]
+      },
+      "evidence":{
+         "type":"array",
+         "items":{
+            "type":"object",
+            "properties":{
+               "kind":{
+                  "type":"string"
+               },
+               "checks":{
+                  "type":"array",
+                  "items":{
+                     "type":"string"
+                  }
+               }
+            }
```

Comment:
@nitro-neal should `kind` or `checks` be in `required` here? The `evidence` array isn't required, but if it does has an object, should that object require the existence of `kind` and/or `checks`? Secondarily, should we set `minItems` to `1` such that in the case `evidence` does exist, then it isn't empty?

cc @kirahsapong

Reply:
Also secondary to this, should we define the `types` here as well? https://www.w3.org/TR/vc-data-model/#types

`"types": ["VerifiableCredential", "KnownCustomerCredential"]`

Reply:
`KnownCustomerCredential` should not be a type - it is not a valid JSON LD type
using json schema makes this superfluous